package.json

```diff
@@ -17,7 +17,8 @@
     "site": "rimraf tmp/site && webpack --env.site --env.element=blaze-elements",
     "commit": "git-cz",
     "commitmsg": "validate-commit-msg",
-    "precommit": "yarn ts:lint && yarn test",
+    "precommit": "npm run ts:lint",
```

Comment:
it's good sanity check to not commit code which is breaking our tslint rules ( check is quick - not like with tests )

yes you can still do no-verify and for that there is tslint check again on prepush.